<launch>
    <node name="apm" pkg="mavros" type="mavros_node">
        <remap to="/heading" from="/apm/global_position/compass_hdg"/>
        <remap to="/control" from="/apm/rc/override"/>
        <remap to="/rcout" from="/apm/rc/out"/>
        <remap to="/power" from="/apm/battery"/>
        <param name="target_system_id" value="1" />
        <param name="target_component_id" value="1" />
        <param name="gcs_url" value="udp://:14556@127.0.0.1:14550" />
    </node>

    <node name="helm" pkg="zoidberg_nav" type="navigation_server.py" respawn="true">
        <param name="controller_frequency" value="10.0" />
        <param name="depth/P" value="50.0" />
        <param name="depth/Pmax" value="200." />
        <param name="depth/tolerance" value=".1" />
        <param name="heading/P" value="2.0" />
        <param name="heading/Pmax" value="200." />
        <param name="heading/tolerance" value="2." />
        <param name="vel/xdiffmax" value="200" />
        <param name="vel/ydiffmax" value="200" />

    </node>

    <node name="pext" pkg="zoidberg_nav" type="pressure_node.py" respawn="true" >
    </node>

    <node pkg="rosbag" type="record" name="zoidberg_bag"
              args="-o zoidberg_bag /rcout /control /depth /heading /power"
    />
</launch>
